app/src/main/assets/language_key_texts/kn.txt

```diff
@@ -1,37 +1,60 @@
 [popup_keys]
-ೌ ಔ %
-ೈ ಐ %
-ಾ ಆ %
-ೀ ಈ %
-ೂ ಊ %
-ಬ ಭ %
-ಹ ಙ %
-ಗ ಘ %
-ದ ಧ %
-ಜ ಝ %
-ಡ ಢ
-ೋ ಓ
-ೇ ಏ
-್ ಅ
-ಿ ಇ
-ು ಉ
-ಪ ಫ
-ರ ಱ ೃ
-ಕ ಖ
-ತ ಥ
-ಚ ಛ
-ಟ ಠ
-ೆ ಒ
-ಂ ಎ
-ನ ಣ
-ಲ ಳ
-ಸ ಶ
-ಋ ್ರ
-ಷ ಕ್ಷ
-ಯ ಜ್ಞ
+ಅ %
+ಆ ಾ %
+ಇ ಿ %
+ಈ ೀ %
+ಉ ು %
+ಊ ೂ %
+ಋ ೄ ೃ ೠ %
+ಎ ೆ %
+ಏ ೇ %
+ಐ ೖ ೈ %
+ಒ ೊ
+ಓ ೋ
+ಔ ೌ
+ಂ ಽ ಼ ಃ ೕ
+ಕ ಕ್ಮ ಕ್ಗ ಕ್ಬ ಕ್ಟ್ರ ಕ್ಷ್ಯ ಕ್ಸ ಕ್ನ ಕ್ವ ಕ್ಲ ಕ್ಯ ಕ್ಷ್ಮ ಕ್ಟ ಕ್ತ ಕ್ಕ ಕ್ಷ ಕ್ರ
+ಖ ಖ್ಹ ಖ್ಖ ಖ್ಗಖ್ಯ ಖ್ವ ಖ್ತ
+ಗ ಗ್ಳ ಗ್ಲ್ಯ ಗ್ಸ ಗ್ರ್ಯ ಗ್ಬ ಗ್ಡ ಗ್ದ ಗ್ತ ಗ್ಧ ಗ್ವ ಗ್ನ ಗ್ಯ ಗ್ಗ ಗ್ರ ಗ್ಲ
+ಘ ಘ್ರ ಘ್ನ
+ಙ ಙ್ಮ
+ಚ ಚ್ಕ ಚ್ಮಚ್ಸ ಚ್ಗ ಚ್ಪ ಚ್ಛ್ರ ಚ್ಹ ಚ್ನ ಚ್ತ ಚ್ಡ ಚ್ಯ ಚ್ಛ ಚ್ಚ
+
+ಛ ಛ್ಘ
+ಜ ಜ್ಪ ಜ್ಗ ಜ್ಬ ಜ್ರ ಜ್ಕ ಜ್ಮ ಜ್ನ ಜ್ಯ ಜ್ಞ ಜ್ಜ ಜ್ವ
+ಝ ಝ್ಕ ಝ್ಡ
+ಞ ಞ್ಞ
+ಟ ಟ್ಜ ಟ್ದ ಟ್ಪ ಟ್ಮ ಟ್ರ್ಯ ಟ್ಸ್ಪ ಟ್ಸ್ಮ ಟ್ಬ ಟ್ಸ ಟ್ಲ ಟ್ರ ಟ್ಗ ಟ್ನ ಟ್ವ ಟ್ಟ ಟ್ರ ಟ್ಯ
+ಠ ಠ್ಮ ಠ್ಯ ಠ್ಠ
+ಡ ಡ್ರ್ಯ ಡ್ಶ ಡ್ಸ ಡ್ಳ ಡ್ಬ ಡ್ವ ಡ್ಮ ಡ್ದ ಡ್ಲ ಡ್ಕ ಡ್ನ ಡ್ಗ ಡ್ತ ಡ್ಡ ಡ್ರ ಡ್ಯ
+ಢ ಢ್ಯ
+ಣ ಣ್ಬ ಣ್ಕ ಣ್ನ ಣ್ಗ ಣ್ರ ಣ್ವ ಣ್ತ ಣ್ಮ ಣ್ಯ ಣ್ಣ
+ತ ತ್ಬ ತ್ಚ ತ್ದ ತ್ಗ ತ್ನ ತ್ವ ತ್ಕ ತ್ಲ ತ್ಪ್ರ ತ್ತ್ವ ತ್ರ್ಯ ತ್ಮ ತ್ಸ ತ್ಪ ತ್ಯ ತ್ರ ತ್ತ
+
+ಥ ಥ್ಯ ಥ್ರ ಥ್ಲ ಥ್ವ ಥ್ರ್ಯ
+ದ ದ್ರ್ಯ ದ್ಳ ದ್ನ ದ್ಮ ದ್ದ ದ್ರ ದ್ಗ ದ್ತ ದ್ಕ ದ್ಲ ದ್ಯ ದ್ಧ ದ್ಷ ದ್ಸ ದ್ಹ ದ್ಬ ದ್ವ
+ಧ ಧ್ಬ ಧ್ಪ ಧ್ಗ ಧ್ಭ‌ ಧ್ಧ ಧ್ವ ಧ್ಯ ಧ್ರ
+ನ ನ್ದ ನ್ಪ ನ್ಡ ನ್ಶ ನ್ಫ ನ್ರ ನ್ಕ ನ್ಸ್ಟ ನ್ಗ ನ್ಲ ನ್ಟ ನ್ಸ್ಪ ನ್ಸ ನ್ಮ ನ್ನ ನ್ಯ ನ್ವ
+ಪ ಪ್ಣ ಪ್ಕ ಪ್ಮ ಪ್ಟ್ನ ಪ್ಗ ಪ್ವ ಪ್ಸ್ಟ ಪ್ಡ ಪ್ಸ ಪ್ಟ ಪ್ನ ಪ್ಲ್ಯ ಪ್ಯ ಪ್ತ ಪ್ರ ಪ್ಪ ಪ್ಲ
+ಫ ೞ ಫ್ಬ ಫ್ನ ಫ್ಸ ಫ್ಲ್ಯ ಫ್ಟ ಫ್ಘ ಫ್ಜ ಫ್ಲ ಫ್ಯ ಫ್ರ ಫ್ತ ಫ್ಟ್ವ
+ಬ ಬ್ತ ಬ್ರ್ಯ ಬ್ಗ ಬ್ಭ ಬ್ಜ ಬ್ನ ಬ್ವ ಬ್ಳ ಬ್ಲ್ಯ ಬ್ಲ ಬ್ಸ ಬ್ಧ ಬ್ಯ ಬ್ಬ ಬ್ರ ಬ್ದ
+ಭ ಭ್ಯ ಭ್ರ
+ಮ ಮ್ವ ಮ್ಡ ಮ್ಸ ಮ್ನ ಮ್ಮ ಮ್ಯ ಮ್ಗ ಮ್ಚ್ಯ ಮ್ತ ಮ್ಕ ಮ್ರ ಮ್ಹ ಮ್ಜ ಮ್ದ ಮ್ಚ ಮ್ಥ ಮ್ಲ
+ಯ ಯ್ಶ ಯ್ಲ್ಯ ಯ್ಜ ಯ್ಪ ಯ್ಬ ಯ್ಗ ಯ್ರ ಯ್ನ ಯ್ವ ಯ್ಡ ಯ್ಸ ಯ್ಲ ಯ್ತ ಯ್ದ ಯ್ಕ ಯ್ಯ
+
+ರ ರ್ಟ ರ್ಧ ರ್ಚ ರ್ಪ ಱ ರ್ಣ ರ್ದ ರ್ಗ ರ್ಭ ರ್ಜ ‌ರ್ಷ ರ್ತ ರ್ಕ ರ್ನ ರ್ಶ ರ್ ರ್ಯ ರ್ಮ ರ್ಥ ರ್ವ
+ಲ ಲ್ಶ ಲ್ರ ಲ್ಜ ಲ್ಡ ಲ್ಫ ಲ್ಯ ಲ್ಲ ಲ್ಮ ಲ್ತ ಲ್ಬ ಲ್ಗ ಲ್ಪ ಲ್ಟ ಲ್ದ ಲ್ಕ ಲ್ಸ ಲ್ನ ಲ್ವ
+ವ ವ್ಮ ವ್ಡ ವ್ಶ ವ್ಚ ವ್ಕ ವ್ತ ವ್ಪ ವ್ಟ ವ್ಲ ವ್ಗ ವ್ಳ ವ್ದ ವ್ಹ ವ್ವ ವ್ಯ ವ್ರ ವ್ನ
+ಶ ಶ್ಶ ಶ್ಕ ಶ್ಚ ಶ್ನ ಶ್ಟ ಶ್ಬ ಶ್ಲ ಶ್ರ ಶ್ವ ಶ್ಮ ಶ್ಣ ಶ್ಟ್ರ ಶ್ಗ ಶ್ಯ
+ಷ ಷ್ರ ಷ್ಗ ಷ್ವ ಷ್ಬ ಷ್ಕ್ರ ಷ್ಟ್ಯ ಷ್ನ ಷ್ಪ್ರ ಷ್ಮ ಷ್ಠ ಷ್ಕ ಷ್ಪ ಷ್ಯ ಷ್ಟ ಷ್ಟ್ರ ಷ್ಣ
+ಸ ಸ್ಖ ಸ್ಗ ಸ್ಡ ಸ್ತ್ರ ಸ್ಟ್ರ ಸ್ಮ ಸ್ಲ ಸ್ರ ಸ್ನ ಸ್ಕ ಸ್ಪ ಸ್ಟ ಸ್ತ್ರ ಸ್ಸ ಸ್ವ ಸ್ಥ ಸ್ತ ಸ್ಯ
+ಹ ಹ್ಸ ಹ್ಞ ಹ್ರ ಹ್ಹ ಹ್ಲ ಹ್ತ ಹ್ಳ ಹ್ವ ಹ್ನ ಹ್ಮ ಹ್ಯ
+ಳ ಳ್ರ ಳ್ಲ ಳ್ಳ್ಯ ಳ್ನ ಳ್ದ ಳ್ಗ ಳ್ಕ ಳ್ಬ ಳ್ಮ ಳ್ಯ ಳ್ವ ಳ್ತ ಳ್ಳ
 
 [labels]
 alphabet: ಅಆಇ
+symbol: ?೧೨೩
+period: .
```

Comment:
`period: .` should not be necessary, as it's the default. Does it work when you remove this line?